test/floatfuncs.jl

```diff
@@ -100,3 +100,13 @@ end
     @test round(Float16(0.6), sigdigits=2) === Float16(0.6)
     @test round(Float16(1.1), sigdigits=70) === Float16(1.1)
 end
+
+@testset "approximation" begin
+
+    @test isapprox(1, 2; atol=1)
```

Comment:
This doesn't test the added method

Reply:
`isapprox` doesn't seem to be tested anywhere else, so I figured I'd ass a test for that as well.  Can remove if needed.

Reply:
It's tested extensively by virtue of being used to test other things. It's even special-cased in the `@test` macro (in the `≈` form) to pass keyword arguments.

I'd keep the focus of this test set on the curried form.

Reply:
Done...sorry this went stale for a bit.  Will try to rebase on current master.